src/import/base.ts

```diff
@@ -30,6 +30,11 @@ export interface ImportOptions {
    * k8s imports will add a "Kube" prefix by default.
    */
   readonly classNamePrefix?: string;
+
+  /**
+   * Path that contains the cdk8s.yaml output file.
+   */
+  readonly configFilePath?: string;
```

Comment:
Not sure I understand where this is being used...our config file path isn't configurable - can you clarify?